modules/ROOT/pages/configure-service-uri-task.adoc

```diff
@@ -0,0 +1,19 @@
+= Configuring Service URI
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When using a load balancer in front of a RAML or OAS API implementation, you can enable the *Keep RAML/OAS base URI* flag. This ensures that the URL that you specify in the proxy matches the one of your API.
+
+// TODO Add an image pointing toi the property in the configuration if necessary
+
+If you prefer to edit the code, then make sure to set the `keepRamlBaseUri` to `true`:
```

Comment:
keepApiBaseUri instead of keepRamlBaseUri

Reply:
Thanks!